src/pages/edge/adobe-journey-optimizer/content-card-ui/Android/tutorial/customizing-content-card-templates.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Customizing Content Card Templates style
+description: This document contains a tutorial on how to customize content card templates style.
+keywords:
+- Adobe Journey Optimizer
+- Guide
+- Messaging
+- Tutorial
+- Content Card
+- UI
+- Android
+---
+
+# Customizing Content Card Templates style
+
+This tutorial explains how to customize the UI of content cards in your application.
+
+## Overview
+
+The Messaging extension provides a way to customize content cards by using [Compose modifiers](https://developer.android.com/develop/ui/compose/modifiers?hl=en) in conjunction with additional parameters provided when creating a style object. Refer to each individual style class to see the customizable settings available for each style.
+
+## Setting custom style parameters when creating a `SmallImageUIStyle`
+
+Perform the following steps to customize content card templates:
+
+1. Create a style object with a `Modifier` as well as any additional customizations passed in as part of the constructor parameters (e.g. if customizing an [AepButtonStyle](../public-classes/styles/aepbuttonstyle.md) , then along with the `Modifier`, the enabled state, elevation, shape, border, color, and padding can be customized if desired.)
```

Comment:
This sentence is off. Perhaps:

1. Create a style object with a `Modifier` passing in any additional customizations as part of the constructor parameters. If customizing an [AepButtonStyle](../public-classes/styles/aepbuttonstyle.md) , for example,  you may pass customizations such as the enabled state, elevation, shape, border, color, and padding alongside the `Modifier`.